zeebe/util/src/main/java/io/camunda/zeebe/util/TlsConfigUtil.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.X509Certificate;
+import java.util.Arrays;
+
+public class TlsConfigUtil {
+  public static void validateTlsConfig(
+      final File certificateChain, final File privateKey, final File keyStore) {
+    if ((certificateChain != null || privateKey != null) && keyStore != null) {
+      throw new IllegalArgumentException(
+          String.format(
+              """
+                      Cannot provide both separate certificate chain and or private key along with a
+                      keystore file, use only one approach.
+                      certificateChainPath: %s
+                      privateKeyPath: %s
+                      OR
+                      keyStorePath: %s""",
```

Comment:
:wrench: Something I missed last time, we have some conventions for error messages: https://github.com/camunda/camunda/wiki/Error-Guidelines

So this would be reworded to something like `Expected to configure with a certificate and private key pair, or with a key store and password, but both were provided. Please select only one approach.`